From d175a7a9f36b25cb768bfbf72d39c9413af92ac5 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Mon, 6 Jul 2020 21:25:45 +0800
Subject: [PATCH] ARM: dts: rv11xx-evb-v10: enable dsi logo display

This patch enable the logo function of DSI.
At the same time, when the panel is initialized in uboot,
We need set the power boot on to avoid black screen.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: I258b395ca676b8a3b41cd23e5db442c521256063
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 1c5c7767049d..aa459adcb4d8 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -144,6 +144,7 @@
 		regulator-name = "vcc18_lcd_n";
 		gpio = <&gpio3 RK_PA7 GPIO_ACTIVE_HIGH>;
 		enable-active-high;
+		regulator-boot-on;
 	};
 
 	vcc5v0_sys: vccsys {
@@ -1246,6 +1247,10 @@
 	>;
 };
 
+&route_dsi {
+	status = "okay";
+};
+
 &saradc {
 	status = "okay";
 	vref-supply = <&vcc_1v8>;
-- 
2.35.3

